charts/incubator/checkmk/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+appVersion: "0.0.1"
```

Comment:
this is where the version of the app should be added, this what the user will see

Reply:
Fixed.

Reply:
> this is where the version of the app should be added, this what the user will see

Not really important to "fix", as it gets updated automagically from CI :P